storage/objects.go

```diff
@@ -11,6 +11,7 @@ import (
 	"time"
 
 	"github.com/joyent/triton-go/client"
+	tt "github.com/joyent/triton-go/errors"
```

Comment:
Following up on previous comments:

a `storage.Error` could embed `triton-go/errors.CommonError` (or some standard package that implements the standard `TritonError` `interface`).